charts/prometheus-opencost-exporter/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+appVersion: 1.106.2
+version: 0.1.0
+description: Prometheus OpenCost Exporter
+home: https://github.com/prometheus-community/opencost_exporter
```

Comment:
This repository does not exist. Will this be created?

Reply:
Updating to https://github.com/opencost/opencost-helm-chart and we'll incorporate this over there

Reply:
but were comes the exporter from?

Reply:
It's from the documentation at https://www.opencost.io/docs/integrations/opencost-exporter and the existing general-purpose OpenCost Helm chart: https://github.com/opencost/opencost-helm-chart. I've opened an issue to host it there: https://github.com/opencost/opencost-helm-chart/issues/128